community-nightlies/rpm96.sh

```diff
@@ -39,6 +39,13 @@ pgdg_check ()
     echo -n "Installing pgdg96 repo... "
 
     yum install -d0 -e0 -y "${repo_url}"
+
+    if ! yum info -q postgresql13-server &> /dev/null; then
```

Comment:
`(deb|rpm)96.sh` scripts are for installing PG 9.6 and supported extensions.

AFAIK, they are outdated and can be removed. It does not makes sense to me that we are checking for PG13 packages for a script that aims to install PG9.6 repos.

Let's keep these files as is, and maybe remove them in a seperate PR.